src/librustc/session/config.rs

```diff
@@ -1036,6 +1036,8 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
           "run all passes except translation; no output"),
     treat_err_as_bug: bool = (false, parse_bool, [TRACKED],
           "treat all errors that occur as bugs"),
+    macro_backtrace: bool = (false, parse_bool, [UNTRACKED],
```

Comment:
could you use `full_macro_backtrace` or something? The name is a bit confusing as is because we get a macro backtrace in any case.

Reply:
Sure. Or maybe `external_macro_backtrace` since you do get a full one for local macros?

Reply:
sure, works for me